monitor.c

```diff
@@ -1888,8 +1893,10 @@ monitor_openfds(struct monitor *mon, int do_logfds)
 		FD_CLOSEONEXEC(pair[1]);
 		mon->m_log_recvfd = pair[0];
 		mon->m_log_sendfd = pair[1];
-	} else
+	}
+	else {
```

Comment:
minimize the differences. Let our code be in sync with upstream code to avoid future merge conflicts.